etc/tendrl/tendrl.conf.sample

```diff
@@ -0,0 +1,17 @@
+[bridge_common]
```

Comment:
should we have separate conf  files for different components like node_agent, and bridges. so that during rpm installation we can copy them to respective locations. Instead of having all the details in sample files of all the conf files.
Just a thought.